docs/src101/src101.md

```diff
@@ -0,0 +1,179 @@
+    "sym": "SIC", //(string)symbol, must be 1-8 characters in length, only English alphabet is available. Blank is not allowed.
+    "pri": "10000", //(uint64)price in sats, must pay to "rec"
+    "desc": "The first src-101. We love btc stamp.", //(string)description for the collection.[optional, default=""]
+    "mintstart": "1706866958", // Unix timestamps in Milliseconds. Mint is available from this time [optional, default="deploy" transaction confirmed blocktime]
+    "mintend": "1710317544", // Unix timestamps in Milliseconds. Mint is not available after this time [optional, default= "18446744073709551615"]
+    "wlf": true, //(bool)whitelist flag.[optional, default=false]
+    "wll": "https://xxxxx", //(string)whitelist link, must be in format of csv, with two params in each line:(string)bitcoin address and (unit64)limit for whitelist mint with discount. This works only when "wlf" is true.[optional, default=""]
+    "disc": "20", //(uint8)discount percentage, must be 1-100, 100 means free mint, 0 means original price. This works only when "wlf" is true.[optional, default="0"]
+    "slf": false //(bool)source link flag, when it's false, it stores images and attributes on BTC network. When it's true, it stores these by metadata link outside of BTC network.[optional, default=false]
+}
+```
+`wll` example: [whitelist.csv](./whitelist.csv)
+
+### MINT
+```JSON
+{
+    "p": "src-101", //(string)protocol standard for non-fungible token
+    "op": "mint", //(string)function name
+    "hash": "0x38091b803f794e50dcc10a9091becaf4f65d35d3ef9e71cfa90c7936af50757e", //(hash256)txid of the deploy transaction
+    "random": false, //(bool)When "random" is false, tokenid is necessary, "amt" is always 1. When "random" is true, "amt" is necessary, tokenid is "";
+    "toaddress": "bc1q7rwd4cgdvcmrxm27xfy6504jwkllge3dda04ww", // recipient address of this mint, can be different from the address in vout[0];
+    "tokenid": "7375706572626f79", //(string)UTF8-> hexstring: superboy->7375706572626f79. It only works when "random" is false [optional, default= "", means random]
+    "amt": "2" // (uint64)mint amount per transaction, can't be more than "lim" of "deploy".It only works when "random" is true.The generated tokenid will follow "the first come the first served" rule: the earlier be persisted on BTC network, the smaller tokenid will be generated. For example, if "max" of "deploy" is 100 and the latest NFT's tokenid is "90", this mint will generate 2 NFT with tokenid "91" and "92". If "max" of "deploy" is "100" and the latest NFT's tokenid is "99", this mint will only generate 1 NFT with tokenid "100" [optional, default= "1"]
+}
+```
+
+### TRANSFER
+```JSON
+{
+    "p": "src-101", //(string)protocol standard for non-fungible token
+    "op": "transfer", //(string)function name
+    "hash": "0x38091b803f794e50dcc10a9091becaf4f65d35d3ef9e71cfa90c7936af50757e", //(hash256)txid of the deploy transaction
```

Comment:
If my amt is 2 in a mint transaction, does that mean I need to transfer both tokens?

Reply:
> If my amt is 2 in a mint transaction, does that mean I need to transfer both tokens?

amt is 2 means it supports mint 2 random NFT in 1 mint transaction("random" must be true, otherwise this will be invalid), it can greatly reduce btc miner fee. transfer is an independent behavior, it should be 1 NFT by 1 tx, can't be 2 NFT in 1 tranfer method.